src/modules/projects/components/ExternalSubmissionsViewModal.tsx

```diff
@@ -96,7 +95,7 @@ const ExternalSubmissionsModalContent = ({
 
   const deleteButton = useMemo(
     () =>
-      submission && (
+      submission.status === ExternalFormSubmissionStatus.New && (
```

Comment:
Suggestion: Hide the delete button if the submission has already been reviewed and converted to a client/enrollment.

Question: If we do this at all, should we do it for all submissions, or only ones that correspond to an enrollment?

Reply:
This seems ok for all submissions. It looks like we don't have acts_as_paranoid on the form submissions, meaning they can't be recovered if deleted. So probably makes sense to restrict it as you suggest